try:
    hoge
except Exception as e:
    print(e)
